index.html

```diff
@@ -34,37 +38,52 @@ <h3>I'am software developer</h3>
                                 alt="Medium" /></a></li>
                 </ul>
             </div>
+            <div class="text-header">
+                <h1>Hey There. I’m Regiss</h1>
+                <h3>I'am software developer</h3>
+                <p>I can help you build a product. feature or website look through some of my work and experience! if
+                    you
+                    like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
         </div>
     </header>
-    <section class="container">
+    <section id="portfolio" class="container">
```

Comment:
- [ ] You have done a great job 👍, But the desktop version of the works section needs improvement. You must stick to the Figma design as much as you can (layouts, margins, font sizes, image sizes ...).
- Your design:
<img width="576" alt="image" src="https://user-images.githubusercontent.com/77190203/154668059-4bbef6f6-c8a5-4b5f-9267-0e9e862137c7.png">

- Figma design:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/77190203/154668267-ccfd203b-249b-4b60-bded-46a3fa8c1c74.png">

- Your design:
<img width="575" alt="image" src="https://user-images.githubusercontent.com/77190203/154668403-dcc33404-56c8-48d1-91d8-ba34da41081d.png">

- Figma design
<img width="395" alt="image" src="https://user-images.githubusercontent.com/77190203/154668535-42898096-e8c7-48c8-bd2f-7447992689ca.png">

- [ ] Kindly fix the `works` section in the desktop version, after that, make sure that the `works` section in the mobile version is not broken.

Reply:
Hi @Regiss05, the Multi-Post Stories part has not been implemented. Let me reach out to you to ensure its is fixed